pkg/koordlet/statesinformer/states_noderesourcetopology.go

```diff
@@ -230,6 +232,14 @@ func (s *nodeTopoInformer) calcNodeTopo() (map[string]string, error) {
 		return nil, fmt.Errorf("failed to marshal cpu manager policy, err: %v", err)
 	}
 
+	// handle cpus reserved by annotation of node.
+	node := s.nodeInformer.GetNode()
```

Comment:
seems that we need some ut for these logic and functions :)

Reply:
done